jib-cli/src/main/java/com/google/cloud/tools/jib/cli/war/WarFiles.java

```diff
@@ -77,12 +77,24 @@ private static List<String> computeEntrypoint(
       CommonContainerConfigCliOptions commonContainerConfigCliOptions)
       throws InvalidImageReferenceException {
     List<String> entrypoint = commonContainerConfigCliOptions.getEntrypoint();
+    String baseImage = fetchBaseImage(commonContainerConfigCliOptions);
```

Comment:
Can this line move to inside the if statement on line 84?

Reply:
Thanks! Removed this line as it is no longer needed after [b893521](https://github.com/GoogleContainerTools/jib/pull/4216/commits/b893521a465b91b2cb73d8bb44e8d9454d3c465e).